### PR TITLE
Update dust shortwave optics for RRTMGP and retuned emis factor for V2

### PR DESCRIPTION
**Pull request by yfenganl**
Update dust shortwave optics file for RRTMGP
Update dust emission factor retuned for V2 for all compsets (from 0.95 to 1.38)

[Non-BFB] - Non Bit-For-Bit
[NML] - Namelist Changing

### CODE REVIEW

**Comment by yfenganl** [comment]
This PR is to add the dust optical updates to RRTMGP. The design document on the dust updates is at: https://acme-climate.atlassian.net/wiki/spaces/ED/pages/1003520636/W12+Improved+Dust+Aerosol+Physics+Design+Document

File: components/cam/bld/namelist_files/namelist_defaults_cam.xml [hunk 1]
```diff
@@ -1077,7 +1077,7 @@
 
 <!-- dust emission tuning factor -->
 <dust_emis_fact                                                         >2.76D0</dust_emis_fact>
-<dust_emis_fact          hgrid="ne30np4"   phys="cam5"                  >0.95D0</dust_emis_fact>
+<dust_emis_fact          hgrid="ne30np4"   phys="cam5"                  >1.38D0</dust_emis_fact>
```
Comment (rljacob) [inline components/cam/bld/namelist_files/namelist_defaults_cam.xml:1080]: what is "hgrid" when pg2 is used?  Not sure this will be picked up.

Comment (yfenganl) [reply components/cam/bld/namelist_files/namelist_defaults_cam.xml:1080]: It looks like pg2 uses the same hgrid settings, but adds another keyword "npg='2'"

**Comment by wlin7** [comment]
Hi @yfenganl , I should let you know before you started working this PR. In @golaz 's recent EAMv1like simulations, we are actually using dust_emis_fact=1.27D0. But those simulation give a higher annual AODVIS at 0.145. So a value larger than 1.27 may be needed anyway. Furthermore, while it is ok to make a change to dust_emis_fact in the many use_case files because of the new dust module, it is almost certain they will need to be retuned.

**Comment by wlin7** [comment]
Just did a calculation of global annual  mean of AODSUST for 1981-2000 for [Chris's recent run of EAMv1like-CLUBBv2. ](https://acme-climate.atlassian.net/wiki/spaces/EWCG/pages/1429962994/20200501.EAMv1like2-CLUBBv2.amip.ne30pg2+r05+oECv3.compy). AODVIS is 0.143, AODSUST 0.0318. Note that AODDUST not available in the default global mean table of e3sm_diags.

**Comment by yfenganl** [comment]
> All changes are fine, unless wanting to start with a different dust_emis_fact value. The value likely needs to be re-tuned anyway.

With the current dust emission scheme, unless the user cases change the surface wind fields (i.e., model resolution, nudged vs free model running), dust emissions are not affected significantly

**Comment by yfenganl** [comment]
> Just did a calculation of global annual mean of AODSUST for 1981-2000 for [Chris's recent run of EAMv1like-CLUBBv2. ](https://acme-climate.atlassian.net/wiki/spaces/EWCG/pages/1429962994/20200501.EAMv1like2-CLUBBv2.amip.ne30pg2+r05+oECv3.compy). AODVIS is 0.143, AODSUST 0.0318. Note that AODDUST not available in the default global mean table of e3sm_diags.

It looks ok. We usually tune AODDUST to be 0.026~0.03

**Comment by yfenganl** [comment]
>Hi @yfenganl , I should let you know before you started working this PR. In @golaz 's recent EAMv1like simulations, we are actually using dust_emis_fact=1.27D0. But those simulation give a higher annual AODVIS at 0.145. So a value larger than 1.27 may be needed anyway. Furthermore, while it is ok to make a change to dust_emis_fact in the many use_case files because of the new dust module, it is almost certain they will need to be retuned. 

Thanks Wuyin @wlin7 That's good to know. I will make a note on the "Dust emission retuning" page

**Comment by golaz** [comment]
@singhbalwinder : is this now ready for merging? Thanks

**Comment by singhbalwinder** [comment]
Thanks @golaz. It is next on my list. Since its Friday, it is not advised to merge NBFB PRs as it won't be possible to bless the diff until Monday. If dashboard is in good shape on Monday, I will merge this PR.

**Pull request merged**